web_responsive/views/web.xml

```diff
@@ -102,7 +103,9 @@
 
         <xpath expr="//a[@class='oe_logo']" position="replace" />
 
-        <xpath expr="//div[@class='oe_footer']" position="replace" />
+        <xpath expr="//div[@class='oe_footer']" position="attributes">
+            <attribute name="style">display: none;</attribute>
```

Comment:
Is there a way to append into the existing class? The goal here is to allow compatibility with addons that may target the div. At the moment it's `oe_footer`, but I don't know if other modules inject classes into that or other things.